mlflow/tracing/export/mlflow_v3.py

```diff
@@ -44,16 +49,83 @@ def export(self, spans: Sequence[ReadableSpan]):
 
         Args:
             spans: A sequence of OpenTelemetry ReadableSpan objects passed from
-                a span processor. Only root spans for each trace should be exported.
+                a span processor. All spans (root and non-root) are exported.
+        """
+        manager = InMemoryTraceManager.get_instance()
+
+        self._export_spans_incrementally(spans, manager)
+        self._export_traces(spans, manager)
+
+    def _export_spans_incrementally(
+        self, spans: Sequence[ReadableSpan], manager: InMemoryTraceManager
+    ):
```

Comment:
```suggestion
    def _export_spans_incrementally(
        self, spans: Sequence[ReadableSpan], manager: InMemoryTraceManager
    ) -> None:
```

can we the return type?

Reply:
Totally - done :)